Weather/Scenes/TimedForecast/TimedForecastPresenter.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TimedForecastPresenter.swift
+//  Weather
+//
+//  Created by Vladislav Kondrashkov on 10/22/18.
+//  Copyright © 2018 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+protocol TimedForecastPresenter {
```

Comment:
From what i see in this protocol Presenter is doing something wrong. Remember rule of thumb – presenter should not include UIKit. In this case it is harder to make a mistake by doing something UI related inside the presenter.

Presenter should not configure UI by the call from somebody. Presenter should call something like `view.display(weather: Weather` on the view.